src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -107,73 +114,31 @@ export class CourseDetailLineChartComponent implements OnChanges {
                 .subtract(this.displayedNumberOfWeeks - 1 + this.displayedNumberOfWeeks * -this.currentPeriod - i, 'weeks')
                 .isoWeekday(1)
                 .isoWeek();
-            this.lineChartLabels[i] = prefix + ' ' + currentWeek;
+            this.dataCopy[0].series[i] = {};
+            this.dataCopy[0].series[i]['name'] = prefix + ' ' + currentWeek;
+            this.absoluteSeries[i] = {};
+            this.absoluteSeries[i]['name'] = prefix + ' ' + currentWeek;
         }
         this.chartTime = startDate.isoWeekday(1).format('DD.MM.YYYY') + ' - ' + endDate.isoWeekday(7).format('DD.MM.YYYY');
+        this.dataCopy[0].name = this.amountOfStudents;
     }
 
     switchTimeSpan(index: boolean): void {
         // eslint-disable-next-line chai-friendly/no-unused-expressions
         index ? (this.currentPeriod += 1) : (this.currentPeriod -= 1);
+        this.showsCurrentWeek = this.currentPeriod === 0;
         this.reloadChart();
     }
 
-    private defineChartOptions() {
-        const self = this;
-        this.lineChartOptions = {
-            layout: {
-                padding: {
-                    top: 20,
-                },
-            },
-            responsive: true,
-            hover: {
-                animationDuration: 0,
-            },
-            animation: {
-                duration: 1,
-                onComplete() {
-                    const chartInstance = this.chart,
-                        ctx = chartInstance.ctx;
-                    ctx.textAlign = 'center';
-                    ctx.textBaseline = 'bottom';
+    formatYAxis(value: any) {
+        return value.toLocaleString() + ' %';
```

Comment:
Since we only show numbers (0-100) here changing between languages does not make a difference. So no explicit need here for `toLocaleString` instead of `toString`, was just the first method that came to my mind.